core/templates/pages/exploration-editor-page/exploration-editor-page.component.ts

```diff
@@ -473,9 +477,6 @@ angular.module('oppia').component('explorationEditorPage', {
             GraphDataService.recompute();
             ExplorationWarningsService.updateWarnings();
           }));
-        $scope.$on('initExplorationPage', (unusedEvtData, successCallback) => {
```

Comment:
Should there be a replacement for this?

Reply:
This had actually already been migrated, I think between resolving merge conflicts it snuck in. This is essentially old code that won't be triggered.